massa-models/src/denunciation.rs

```diff
@@ -797,6 +820,28 @@ impl From<&DenunciationPrecursor> for DenunciationIndex {
     }
 }
 
+impl Ord for DenunciationIndex {
+    fn cmp(&self, other: &Self) -> Ordering {
+        let self_key = (
+            self.get_slot(),
+            self.get_index().unwrap_or(&0),
+            u32::from(DenunciationIndexTypeId::from(self)),
```

Comment:
As mentioned in the previous review, this should be commented to explain why the type ID should be kept as the last element of the tuple

Reply:
Added